editor/dependency_editor.cpp

```diff
@@ -271,7 +271,11 @@ DependencyEditor::DependencyEditor() {
 }
 
 /////////////////////////////////////
-void DependencyEditorOwners::_list_rmb_select(int p_item, const Vector2 &p_pos) {
+void DependencyEditorOwners::_list_rmb_clicked(int p_item, const Vector2 &p_pos, MouseButton p_mouse_button_index) {
+	if (p_mouse_button_index == MouseButton::RIGHT) {
```

Comment:
```suggestion
	if (p_mouse_button_index != MouseButton::LEFT) {
```
It should be like this if you want to check for lmb.